Tasks/SqlDacpacDeploymentOnMachineGroup/make.json

```diff
@@ -3,7 +3,7 @@
        "nugetv2": [
             {
                 "name": "TaskModuleSqlUtility",
-                "version": "0.1.0",
+                "version": "0.1.2",
```

Comment:
There are comments on the Utility PR. So this is likely to change post your update on the Utility.